static/app/views/releases/list/index.tsx

```diff
@@ -118,6 +124,24 @@ class ReleasesList extends DeprecatedAsyncView<Props, State> {
       ],
     ];
 
+    if (this.hasV2ReleaseUIEnabled) {
+      const thresholdQuery: ThresholdQuery = {};
+      if (selection.projects.length) {
+        thresholdQuery.project = selection.projects;
+      } else {
+        thresholdQuery.project = [-1];
```

Comment:
Might be too late for this contract, but what is `-1` supposed to help mean here? Could we add some comment here explaining that, or maybe make it a const? Since this is a query, could not passing in this field have been sufficient?

Reply:
Good question

This unfortunately is a relic that we won't ever be cleaning up :( 
`-1` means "all projects"

on the `.py` side we have a [constant ](ALL_ACCESS_PROJECTS), but on the FE we do not :(

Reply:
Ahh that is good information, thank you!

Reply:
Actually - i do see we have a constant defined.
I'll utilize that